components/post_view/post_recent_reactions/recent_reactions_emoji_item.jsx

```diff
@@ -24,18 +24,18 @@ class EmojiItem extends React.Component {
             emoji,
             order,
         } = this.props;
-        const itemClassName = 'post-menu__item';
+        const itemClassName = 'post-menu__item post-menu__emoticon';
```

Comment:
`itemClassName` is reused to generate a test ID on the button below, so I think changing this will break E2E tests that rely on that. Could we just apply this change to the outer div?